apps/system/js/lock_screen_window_manager.js

```diff
@@ -398,21 +388,13 @@
    * @this {LockScreenWindowManager}
    * @memberof LockScreenWindowManager
    */
-  LockScreenWindowManager.prototype.initWindow =
-    function lwm_initWindow() {
-      var req = window.SettingsListener.getSettingsLock()
-        .get('lockscreen.enabled');
-      req.onsuccess = () => {
-        this.states.ready = true;
-        if (true === req.result['lockscreen.enabled'] ||
-           'true' === req.result['lockscreen.enabled']) {
-          this.states.enabled = true;
-        } else if (false === req.result['lockscreen.enabled'] ||
-                   'false' === req.result['lockscreen.enabled']) {
-          this.states.enabled = false;
-        }
-        this.openApp();
-      };
+  LockScreenWindowManager.prototype['_observe_lockscreen.enabled'] =
```

Comment:
I don't...really like this style... It breaks the naming convention of 'methods' and 'property'. Is it necessary?